specs/data_structures.md

```diff
@@ -79,19 +79,19 @@
 
 ## Type Aliases
 
-| name                        | type                       |
-|-----------------------------|----------------------------|
-| [`Address`](#address)       | `byte[32]`                 |
-| `Amount`                    | `uint64`                   |
-| `Graffiti`                  | `byte[MAX_GRAFFITI_BYTES]` |
-| [`HashDigest`](#hashdigest) | `byte[32]`                 |
-| `Height`                    | `uint64`                   |
-| `NamespaceID`               | `byte[NAMESPACE_ID_BYTES]` |
-| `Nonce`                     | `uint64`                   |
-| `Round`                     | `uint64`                   |
-| `StateSubtreeID`            | `byte`                     |
-| [`Timestamp`](#timestamp)   | `uint64`                   |
-| `VotingPower`               | `uint64`                   |
+| name                        | type                        |
+|-----------------------------|-----------------------------|
+| [`Address`](#address)       | `byte[32]`                  |
+| `Amount`                    | `uint64`                    |
+| `Graffiti`                  | `byte[MAX_GRAFFITI_BYTES]`  |
+| [`HashDigest`](#hashdigest) | `byte[32]`                  |
+| `Height`                    | `int64`                     |
```

Comment:
`int64` vs `uint64`? Idk exactly

If there is no difference, changing the specs to conform impl is easier. Otherwise, I can change the impl, like I did in [#315]https://github.com/lazyledger/lazyledger-core/pull/315() also if uin64 is in some way better.  
Same for `Round`